plutus-tx/src/PlutusTx/Lift/Instances.hs

```diff
@@ -52,7 +53,7 @@ instance Typeable uni (->) where
 -- Primitives
 
 typeRepBuiltin
-    :: forall a uni fun. uni `PLC.Includes` a
+    :: forall (a :: GHC.Type) uni fun. uni `PLC.Includes` a
     => Proxy a -> RTCompile uni fun (Type TyName uni ())
 typeRepBuiltin (_ :: Proxy a) = pure $ mkTyBuiltin @a ()
```

Comment:
`mkTyBuiltin` is kind-polymorphic. Should this one be too? Or perhaps we don't need it here? Pinging @michaelpj